fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -498,4 +492,24 @@ object TestHelper {
                 .contains("mInputShown=true"),
         )
     }
+
+    /**
+     * The list of Search engines for the "home" region of the user.
+     * For en-us it will return the 6 engines selected by default: Google, Bing, DuckDuckGo, Amazon, Ebay, Wikipedia.
+     */
+    fun getRegionSearchEnginesList(): List<SearchEngine> {
+        val searchEnginesList = appContext.components.core.store.state.search.regionSearchEngines
```

Comment:
Good to know this works 🥳

Reply:
yes, but it only returns 6 out of 8 engines installed. You also need to use the second method to get them all.